benchmarking/benchmark.py

```diff
@@ -1,8 +1,9 @@
-import numpy as np
-import yaml
 import os
+import re
 import h5py
+import yaml
 import socket
+import numpy as np
```

Comment:
Let's split these, first listing standard lib packages then an empty line and then other packages.

Reply:
Think I moved them to the desired order now.